site/content/en/docs/Third Party Content/examples.md

```diff
@@ -19,3 +19,10 @@ Microsoft Windows, MacOS, and Android. Minetest game play is very similar to tha
   server hosted on Kubernetes using Agones. It wraps the Minetest server with a [Go](https://golang.org) binary, and introspects stdout to provide the event hooks for the SDK integration. The wrapper is from [Xonotic Example](https://github.com/googleforgames/agones/blob/main/examples/xonotic/main.go) with a few changes to look for the Minetest ready output message.  
 
 You will need to download the Minetest client separately to play.
+
+## Quilkin
+
+* [Quilkin](https://github.com/googleforgames/quilkin) is a non-transparent UDP proxy specifically designed for use with large scale multiplayer dedicated game server deployments, to ensure security, access control, telemetry data, metrics and more.
+* [Quilkin with Agones](https://github.com/googleforgames/quilkin/tree/main/examples) is an example of running [Xonotic with Quilkin](https://github.com/googleforgames/quilkin/tree/main/examples/agones-xonotic-sidecar) on an Agones cluster, utilising the sidecar integration pattern and Quilkin xDS Agones provider with a TokenRouter to provide routing and access control to the allocated GameServer instance.
```

Comment:
Tweaking some links, since there are a few options to link to - but otherwise, looks good 👍🏻 

```suggestion
* [Quilkin with Agones](https://github.com/googleforgames/quilkin/tree/main/examples) is an example of running [Xonotic](https://xonotic.org/) with Quilkin on an Agones cluster, utilising either [the sidecar integration pattern](https://github.com/googleforgames/quilkin/tree/main/examples/agones-xonotic-sidecar) or via the the [Quilkin xDS Agones provider](https://github.com/googleforgames/quilkin/tree/main/examples/agones-xonotic-xds) with a TokenRouter to provide routing and access control to the allocated GameServer instance.
```

Reply:
I have modified this line